vmm/src/vm_config.rs

```diff
@@ -596,4 +596,9 @@ pub struct VmConfig {
     pub gdb: bool,
     pub platform: Option<PlatformConfig>,
     pub tpm: Option<TpmConfig>,
+    // Preseved FDs are the ones that share the same life-time as its holding
+    // VmConfig instance, such as FDs for creating TAP devices.
+    // Perserved FDs will stay open as long as the holding VmConfig instance is
+    // valid, and will be closed when the holding VmConfig instance is destroyed.
+    pub preserved_fds: Option<Vec<i32>>,
```

Comment:
Cool, but I think this should be `#[serde(skip)]` I don't think this makes any sense to serialise/deserialise

Reply:
Good catch. Updated.